mlflow/sagemaker/__init__.py

```diff
@@ -2336,8 +2366,9 @@ def update_deployment(
                          Defaults to ``None``.
 
                        - ``variant_name``: A string specifying the desired name when creating a
-                                           production variant.  Defaults to ``None``.
-
+                                           production variant.  Defaults to ``None``.                                           
+                       - ``endpoint_config``: A dictionary specifying the endpoint configuration 
+                        as key-value pairs to be set for the deployed endpoint
```

Comment:
Nit: Should this be `async_inference_config` instead of `endpoint_config` + a different docstring?

Reply:
@dbczumar  Good catch, thanks. I have updated it.